ISSUE-TRIAGE.md

```diff
@@ -0,0 +1,106 @@
+# Triaging of issue
+[The contents have been sourced from [docker/docker project](https://github.com/docker/docker/blob/master/project/ISSUE-TRIAGE.md)]
```

Comment:
My original thought for "linking" was to not include the actual contents here, but only link to the document in the docker repo.

If the content is *copied*, as it is now, I'm not sure we need the link.

@moxiegirl wdyt? Keep it for reference, or remove it?